tests/env/testdata/mismatch.crt

```diff
@@ -0,0 +1,23 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Need some information on how these mismatch certs were generated. I cannot tell what the "mismatch" is looking at this file

Reply:
Mismatched crt is used in the case when the server crt is unknown to the client root ca. Before, we used proxy.crt to replace server.crt or vice versa to make this case but it is confusing.